### DISCUSSION TITLE
limit() and offset() for UnionAll

### DISCUSSION THREAD

**Discussion by toxeus** [diesel-rs/diesel#3084] [Q&A]
Hi, I just wanted to know why `limit()` and `offset()` is not available for the type returned by `union_all()`? Is it a technical limitation or is it just a matter of implementing it?

Thanks!

**Comment by weiznich** [comment]
Its likely just not implemented yet.

**Comment by Aethelflaed** [comment]
I'm giving this a try with https://github.com/diesel-rs/diesel/pull/4245
